CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 # Changelog
+### 2.22.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.12](https://fluentbit.io/announcements/v1.8.12/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.8.1
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.21.6` this release adds:
+* Feature - Add support for external_id in AWS output plugins [aws-for-fluent-bit:171](https://github.com/aws/aws-for-fluent-bit/issues/171)
+* Bug - Fix truncation issue after compression [kinesis:183](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit/issues/183)
```

Comment:
Maybe add * Bug - Fix return value from `tls_net_read` [fluentbit:4098](https://github.com/fluent/fluent-bit/issues/4098) so people know why we stopped cherrypicking? What do you think?

Reply:
yeah I forgot to ask you guys about it. So I was think about it but I think we don't need to add to this field as `2.21.6` already had it. Maybe 
```
Same as `2.21.6` this release includes the following fixes which have already been accepted in upstream:
* bug - xxxx
```

Reply:
Let's please not mention the cherry-pick file directly in the release notes. Release notes are for customers to consume and need to be worded very carefully to be clear and useful and should not make reference to our release process.

Also, I am very confused- if 2.21.6 already had this in upstream, then why are we only removing it from the cherry-pick file now? From what i see Ramya's PR was just merge and 1.8.12 is its first upstream release. https://fluentbit.io/announcements/v1.8.12/
https://github.com/fluent/fluent-bit/pull/4584

Reply:
So, I think that because 1.8.12 is the first release that has Ramya's commit in it, we no longer have to add this commit via cherry pick on top of the base 1.8.12.

We had to cherry pick this onto 1.8.11 because 1.8.11 did not have Ramya's commit.